10up/theme-scaffold
# Add WP Acceptance support and initial tests

For #131.

- Adds 10up/wpacceptance as Composer dev dependency
- Removes 10/wp-mock as Composer dev dependency (wasn't being used)
- Adds Standard WP Functionality Tests
- Adds Frontend Functionality Tests that cover some of the functionality in place in the scaffold
  - Ensures stylesheet, javascript are enqueued
  - Ensures rss links are present
  - Ensures body class is dynamic

In defense of what may seem like low value tests, I think it's important to have the acceptance testing library present with some sample tests setup to serve as an example.

Testing Instructions:
_Requirements: Composer, Docker_
1. Run `composer update` to ensure you have the packages.
2. Run `./vendor/bin/wpacceptance run` from the root of this repository.
3. See tests run. (They should all pass hopefully)
